e2e/svelte/scripts/setupTypeScript.js

```diff
@@ -3,99 +3,104 @@
 
 // Add script for checking
 packageJSON.scripts = Object.assign(packageJSON.scripts, {
-  check: 'svelte-check --tsconfig ./tsconfig.json'
+  "check": "svelte-check"
 })
 
 // Write the package JSON
-fs.writeFileSync(
-  path.join(projectRoot, 'package.json'),
-  JSON.stringify(packageJSON, null, '  ')
-)
+fs.writeFileSync(path.join(projectRoot, "package.json"), JSON.stringify(packageJSON, null, "  "))
 
 // mv src/main.js to main.ts - note, we need to edit rollup.config.js for this too
-const beforeMainJSPath = path.join(projectRoot, 'src', 'main.js')
-const afterMainTSPath = path.join(projectRoot, 'src', 'main.ts')
+const beforeMainJSPath = path.join(projectRoot, "src", "main.js")
+const afterMainTSPath = path.join(projectRoot, "src", "main.ts")
 fs.renameSync(beforeMainJSPath, afterMainTSPath)
 
 // Switch the app.svelte file to use TS
-const appSveltePath = path.join(projectRoot, 'src', 'App.svelte')
-let appFile = fs.readFileSync(appSveltePath, 'utf8')
-appFile = appFile.replace('<script>', '<script lang="ts">')
-appFile = appFile.replace('export let name;', 'export let name: string;')
+const appSveltePath = path.join(projectRoot, "src", "App.svelte")
+let appFile = fs.readFileSync(appSveltePath, "utf8")
+appFile = appFile.replace("<script>", '<script lang="ts">')
+appFile = appFile.replace("export let name;", 'export let name: string;')
 fs.writeFileSync(appSveltePath, appFile)
 
 // Edit rollup config
-const rollupConfigPath = path.join(projectRoot, 'rollup.config.js')
-let rollupConfig = fs.readFileSync(rollupConfigPath, 'utf8')
+const rollupConfigPath = path.join(projectRoot, "rollup.config.js")
+let rollupConfig = fs.readFileSync(rollupConfigPath, "utf8")
 
 // Edit imports
-rollupConfig = rollupConfig.replace(
-  "'rollup-plugin-terser';",
-  `'rollup-plugin-terser';
+rollupConfig = rollupConfig.replace(`'rollup-plugin-css-only';`, `'rollup-plugin-css-only';
 import sveltePreprocess from 'svelte-preprocess';
-import typescript from '@rollup/plugin-typescript';`
-)
+import typescript from '@rollup/plugin-typescript';`)
 
 // Replace name of entry point
-rollupConfig = rollupConfig.replace("'src/main.js'", "'src/main.ts'")
+rollupConfig = rollupConfig.replace(`'src/main.js'`, `'src/main.ts'`)
 
 // Add preprocessor
 rollupConfig = rollupConfig.replace(
   'compilerOptions:',
   'preprocess: sveltePreprocess({ sourceMap: !production }),\n\t\t\tcompilerOptions:'
-)
+);
 
 // Add TypeScript
 rollupConfig = rollupConfig.replace(
   'commonjs(),',
   'commonjs(),\n\t\ttypescript({\n\t\t\tsourceMap: !production,\n\t\t\tinlineSources: !production\n\t\t}),'
-)
+);
 fs.writeFileSync(rollupConfigPath, rollupConfig)
 
-// Add TSConfig
+// Add svelte.config.js
 const tsconfig = `{
   "extends": "@tsconfig/svelte/tsconfig.json",
 
   "include": ["src/**/*"],
   "exclude": ["node_modules/*", "__sapper__/*", "public/*"]
 }`
-const tsconfigPath = path.join(projectRoot, 'tsconfig.json')
+const tsconfigPath =  path.join(projectRoot, "tsconfig.json")
```

Comment:
any extra space got added here
```suggestion
const tsconfigPath = path.join(projectRoot, "tsconfig.json")
```